mito-ai/mito_ai/anthropic_client.py

```diff
@@ -148,51 +149,46 @@ async def request_completions(self, messages: List[ChatCompletionMessageParam],
         """
         Get a response from Claude or the Mito server that adheres to the AgentResponse format.
         """
-        try:
```

Comment:
Instead of catching the exceptions in the clients, we let the exceptions bubble up to the provider.py so we can manage all of the exceptions in one centralized place.